### PR TITLE
[DBAAS-417] tenancy refactor

### PR DESCRIPTION
**Pull request by tchughesiv**
depends on https://github.com/RHEcosystemAppEng/dbaas-operator/pull/175

https://issues.redhat.com/browse/DBAAS-417

Signed-off-by: Tommy Hughes <tohughes@redhat.com>

### CODE REVIEW

**Comment by yzhao583** [comment]
@tchughesiv Looks like there are conflicts

**Comment by tchughesiv** [comment]
@yzhao583 tks

**Review by olavtar — COMMENTED** [review]
ns disabled bug is Fixed.

File: src/components/adminDashboard.jsx [hunk 1]
```diff
@@ -289,20 +302,35 @@ const AdminDashboard = () => {
...
-  React.useEffect(() => {
-    mapDBaaSConnectionsAndServiceBindings()
+  React.useEffect(async () => {
+    await fetchCSV()
+  }, [])
+
+  React.useEffect(async () => {
+    await fetchInstances()
+    await fetchDBaaSConnections()
+    await fetchServiceBindings()
```
[the remark this answers is not in the document]
Comment (tchughesiv) [reply src/components/adminDashboard.jsx]: @yzhao583 thanks for the feedback... this was actually leftover code from my original (unsuccessful) dev efforts. 
many of my early issues were around non-privileged openshift users, folks without cluster-admin rights. for these users the console wasn't properly displaying or reacting to all function results on certain pages.

being newer to React, it took some time for me to discover the useEffect 'dependency array' as a proper way to force necessary updates to occur. once i started leveraging that, everything started to click into place.

I've cleaned up those async references.

**Review by yzhao583 — APPROVED** [review]

**Pull request merged**